packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```
@@ -186,3 +186,13 @@ button.font-library-modal__upload-area {
 		justify-content: center;
 	}
 }
+
+.font-library-modal__select-all {
+	width: 100%;
+	height: auto;
+	padding: $grid-unit-20;
+
+	.components-checkbox-control__label {
+		padding-left: $grid-unit-20;
+	}
+}
```

```suggestion
.font-library-modal__select-all {
	padding: $grid-unit-20 $grid-unit-20 $grid-unit-20 $grid-unit-20 + $border-width;

	.components-checkbox-control__label {
		padding-left: $grid-unit-20;
	}
}
```

To properly align the vertical position of the checkboxes, let's add a 1px border width to the left side.

I've also removed styles that we don't need.

---

Good catch!